internals/overlord/servstate/handlers.go

```diff
@@ -115,13 +118,21 @@ func (m *ServiceManager) doStart(task *state.Task, tomb *tomb.Tomb) error {
 	}
 
 	currentPlan := m.getPlan()
-	config, ok := currentPlan.Services[request.Name]
-	if !ok {
+	config, configFound := currentPlan.Services[request.Name]
+	if !configFound {
```

Comment:
nit: Was the name originally used lower down? Just the rename on its own does not seem to add much value, especially with the error below describing it ?